stf/test/saros/stf/test/stf/menubar/SarosPreferencesTest.java

```diff
@@ -124,14 +135,15 @@ public void editAccount() throws RemoteException {
         ALICE.superBot().menuBar().saros().preferences().isAccountActive(Constants.JID_TO_CHANGE));
   }
 
-  @Test(expected = RuntimeException.class)
   public void deleteActiveAccount() throws RemoteException {
```

Comment:
You removed the `@Test` and the method has not the prefix `test` and the method is not called. Do you want to remove the test?

Reply:
Seems I removed a little bit to much.